modules/simple_forms_api/spec/requests/simple_forms_api/v1/simple_forms_spec.rb

```diff
@@ -875,8 +809,8 @@
 
         context 'non-veteran preparer' do
           it 'successful submission' do
-            allow_any_instance_of(SimpleFormsApi::PdfUploader)
-              .to receive(:upload_to_benefits_intake).and_return([200, confirmation_number])
+            allow_any_instance_of(SimpleFormsApi::V1::UploadsController)
+              .to receive(:upload_pdf).and_return([200, confirmation_number])
```

Comment:
Can the redundant mock logic for `SimpleFormsApi::V1::UploadsController` and `VANotify::EmailJob` be placed in a `before` block?

Reply:
I wouldn't do it for `SimpleFormsApi::V1::UploadsController.upload_pdf` because it returns `200` sometimes and `500` other times. I suppose we could do it for `VANotify::EmailJob.perform_async`. Another team wrote those specs so I'm not so keen on refactoring their work. I'm on the fence about it.

Reply:
What you have here is fine, but if you wanted to reduce redundancy, you could do the following:

```ruby
before do
  allow_any_instance_of(SimpleFormsApi::V1::UploadsController).to receive(:upload_pdf).and_return(response)
end
```

And within the `it` block you could do something like:

```ruby
let(:response) { [200, confirmation_number] }
```

Reply:
I'll go ahead and approve as I don't have strong convictions here but I'd be happy to approve again if you make any changes.

Reply:
Love it! I've implemented this change.